pom.xml

```diff
@@ -135,6 +135,11 @@
             <artifactId>apache-client</artifactId>
             <version>${awssdk.version}</version>
         </dependency>
+        <dependency>
+            <groupId>software.amazon.awssdk</groupId>
+            <artifactId>arns</artifactId>
```

Comment:
I assume that this is where the new functionality comes in to read in the arn. I also assume that it is plural because it can be a list. Is there anywhere where we document clearly that this is the case

Reply:
It is not plural because it can be a list. That is what the artifact is called (and was named by whomever is the creator of software.amazon.awssdk). It refers to the package arns package: https://central.sonatype.com/artifact/software.amazon.awssdk/arns

Reply:
Is there any way we can document this use case. I am confused on how this will surface well in the code to be documented for future developers and users. For example can we comment that link into the pom file?